components/avatar/avatar.vue

```diff
@@ -206,22 +216,27 @@ export default {
     },
 
     randomizeGradientAngle () {
-      return getRandomElement(AVATAR_ANGLES);
+      return getRandomElement(AVATAR_ANGLES, this.seed);
     },
 
     randomizeGradientColorStops () {
       const colors = new Set();
 
+      let count = 0;
       // get 3 unique colors, 2 from colorsWith100 and one from colorsWith200
       while (colors.size < MAX_GRADIENT_COLORS) {
+        // add count to the seed since we are looking for 3 unique colors. If the seed makes it always
+        // return the same color we'll get an infinite loop.
+        const seedForColor = this.seed === undefined ? undefined : this.seed + count.toString();
```

Comment:
nit:

```suggestion
        const seedForColor = this.seed ? this.seed + count.toString() : undefined;
```

Reply:
not quite the same because in this case empty string would become undefined